vg/recorder/recorder.go

```diff
@@ -112,6 +112,11 @@ func (c *Canvas) append(a Action) {
 	c.Actions = append(c.Actions, a)
 }
 
+func (c *Canvas) DPI() float64 {
+	// FIXME(sbinet): add a way to specify a value?
```

Comment:
If we need to add these we can either make canvases require a constructor which takes a DPI, or add a resolution field that is interpreted as 72 if it's 0, or make it a conditional interface where 72 is the default value if there is no DPI method. Given that LaTeX is the only one that needs it, the last approach feels the nicest to me.

Reply:
I am a bit worried a `xyz.Canvas` implementing this `DPI()` method could be wrapped by a couple layers of `vg.Canvas` (sans the `DPI()` method) interfaces and then presented to a `draw.Canvas` value.
I think the type switch wouldn't be able to dig that `DPI()` method out of the interface layers.

but perhaps it's just a remote, theoretical and uncommon failure mode....